src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
```

Comment:
```suggestion
    },
    
    get info() {
```
It would be great to add empty lines for better readability